scout/server/blueprints/dashboard/forms.py

```diff
@@ -16,12 +11,7 @@ def pre_validate(self, form):
         pass
 
 
-class DashboardFilterForm(FlaskForm):
-    """Takes care of cases filtering in cases page"""
+class DashboardFilterForm(CaseFilterForm):
```

Comment:
Inherit the same form as in cases search, just add the select for searching into all institutes or a specific institute